src/main/kotlin/org/rust/lang/core/resolve2/ModCollectorBase.kt

```diff
@@ -501,6 +504,9 @@ data class MacroDefLight(
 data class Macro2DefLight(
     val name: String,
     val visibility: VisibilityLight,
+    val body: String,
+    val bodyHash: HashCode?,
+    val hasRustcBuiltinMacro: Boolean,
```

Comment:
Lets use `hasRustcBuiltinMacro` and `bodyHash` in `writeTo` method. And also add corresponding test to `RsDefMapUpdateChangeSingleFileTest`